src/server/lib/example.js

```diff
@@ -6,3 +6,21 @@ exports.pluralize = function (count, singular, plural) {
   return `${count} ${plural}`;
 };
 
+
+exports.pluralize_v2 = function (count, singular, plural) {
```

Comment:
Codacy found an issue: [Identifier 'pluralize_v2' is not in camel case.](https://app.codacy.com/gh/codacy-acme/nodeproject/pullRequest?prid=5794001)